code/modules/food_and_drink/snacks.dm

```diff
@@ -1118,14 +1120,6 @@
 	initial_reagents = "sugar"
 	food_effects = list("food_energized")
 
-	heal(var/mob/M)
-		if(ishuman(M) && (M.job in list("Security Officer", "Head of Security", "Detective")))
-			src.heal_amt *= 2
-			..()
-			src.heal_amt /= 2
```

Comment:
excuse?

Reply:
I moved that functionality to the foodheal component. I could move the functionality in there back into the item's on_eat proc, though.